bot/cogs/help_channels.py

```diff
@@ -549,6 +571,19 @@ def is_dormant_message(message: t.Optional[discord.Message]) -> bool:
                 log.trace(f"Resetting send permissions for {member} ({member.id}).")
                 await self.available_category.set_permissions(member, send_messages=None)
 
+    async def reset_send_permissions_for_help_user(self, channel: discord.TextChannel) -> None:
+        """Reset send permissions in the Available category for member that started the help session in `channel`."""
+        log.trace(f"Attempting to find user for help session in #{channel.name} ({channel.id}).")
+        try:
+            member = self.help_channel_users[channel]
+        except KeyError:
+            log.trace(f"Channel #{channel.name} ({channel.id}) not in help session cache, permissions unchanged.")
+            return
+        log.trace(f"Resetting send permissions for {member} ({member.id}).")
+        await self.available_category.set_permissions(member, send_messages=None)
+        # Cancel task, ignore no task existing when the claim time passed but idle time has not.
```

Comment:
It took me a while to understand the comment. It could be written more clearly:

```suggestion
        # Ignore missing task when claim cooldown has passed but the channel still isn't dormant.
```